### DISCUSSION TITLE
Is there a way to enable 'brush' button for example while on readyOnly?

### DISCUSSION THREAD

**Discussion by Reinoldo** [xdan/jodit#931] [Q&A] [answered]
Hi people,

I know I can set the button active while on readonly, but while it seems to be active and working, suddently does not.

Can I manipulate this behavior through exec options for instance?
Something like:
```
 buttons: [
    {
      name: "brush",
      exec: (editor) => {
       //do something to enable brush on readyonly
        
      }
    }
  ],
```

**Comment by xdan** [comment, marked as answer]
In readonly the editor blocks all edits by design, so even if you force a button active it can't change the content — that's why brush seems to work and then suddenly doesn't. `activeButtonsInReadOnly` only keeps a button clickable/visible; it doesn't re-enable the editing commands behind it.

If you need the user to actually apply formatting, the editor can't be readonly for that action — toggle readonly off while they do it, or use a normal editor with the other buttons removed instead of readonly.
